cypress/integration/slidingTreeDom.spec.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+Cypress.Commands.add('hideTree', (selector) => {
+  cy.get('ul').contains(selector).click();
+
+  cy.get('ul').contains(selector).parent().should(($ul) => {
```

Comment:
would something like below work
cy.get('ul').contains(selector).children().should('not.be.visible')?

Reply:
nope, it wouldn't. 
cy.get('ul').contains(selector) or cy.contains(selector)  - selects <span> element without children.